src/main.cpp

```diff
@@ -994,15 +994,15 @@ static bool determine_subgame(GameParams *game_params)
 		if (game_params->game_spec.isValid()) {
 			gamespec = game_params->game_spec;
 			infostream << "Using commanded gameid [" << gamespec.id << "]" << std::endl;
-		} else { // Otherwise we will be using "minetest"
-			gamespec = findSubgame(g_settings->get("default_game"));
-			infostream << "Using default gameid [" << gamespec.id << "]" << std::endl;
-			if (!gamespec.isValid()) {
-				errorstream << "Game specified in default_game ["
-				            << g_settings->get("default_game")
-				            << "] is invalid." << std::endl;
-				return false;
+		} else {
+			if (game_params->is_dedicated_server) {
+				// If this is a dedicated server and no gamespec has been specified,
+				// print a friendly error pointing to ContentDB.
+				errorstream << "To run a " PROJECT_NAME_C " server, you need a game." << std::endl
+				            << "Check out https://content.minetest.net for a selection of games to pick from and download." << std::endl;
```

Comment:
this shows up even when you have a game installed but just didn't pick one, needs adjustment

Reply:
nope there isn't.
For me it would suffice if the message just said something like "To run a Minetest server, you need to pick a game (via --gameid)." plus the second line.

Reply:
Ah I see, I probably shouldn't overengineer it then :P